rites/roman1969/src/calendars/countries/brazil/index.ts

```diff
@@ -35,6 +41,12 @@ export class Brazil extends CalendarDef {
       dateDef: { month: 6, date: 15 },
     },
 
+    augustine_zhao_rong_priest_and_companions_martyrs: {
+      // In Brazil, St Augustine Zhao Rong and companions are transferred to July 8
+      // because St Paulina is celebrated on July 9
```

Comment:
```suggestion
      // because St Paulina is celebrated on July 9
      // src: https://arquidiocesemilitar.org.br/wp-content/uploads/2025/01/DIRETORIO-LITURGICO-OMB-2025-1_compressed.pdf#page=85
```